src/main/java/uk/gov/hmcts/reform/sandl/snlapi/config/BeanConfiguration.java

```diff
@@ -0,0 +1,32 @@
+package uk.gov.hmcts.reform.sandl.snlapi.config;
+
+import org.springframework.boot.web.client.RestTemplateBuilder;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.client.ClientHttpResponse;
+import org.springframework.web.client.DefaultResponseErrorHandler;
+import org.springframework.web.client.RestTemplate;
+import uk.gov.hmcts.reform.sandl.snlapi.exceptions.OptimisticLockException;
+
+import java.io.IOException;
+
+@Configuration
```

Comment:
Why Configuration annotate on this is needed?
Why is this in the config package? (it is not similar to other in this area)

Reply:
It is similar to SwaggerConfiguration.